build.gradle

```diff
@@ -15,10 +15,10 @@ repositories {
 }
 
 dependencies {
-    api group: 'org.java-websocket', name: 'Java-WebSocket', version: '1.5.1'
```

Comment:
Would it make sense to keep `Java-WebSocket` as an `api` dependency? Unlike the other two (which are handled internally), WebSocket objects are returned via the public api (ex. `WSConnection.connectHALSim`).

Reply:
Fixed.